portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="gallery">
+      <h1>Here are some pictures that I like</h1>
+
+      <div class="image-thumbnails">
+        <a href="images/image-1.JPG"><img src="images/image-1.JPG"/></a>
+        
+        <a href="images/image-2.JPG"><img src="images/image-2.JPG"/></a>
+        <a href="images/image-3.jpg"><img src="images/image-3.jpg"/></a>
+        <a href="images/image-4.jpg"><img src="images/image-4.jpg"/></a>
+        <a href="images/image-5.jpg"><img src="images/image-5.jpg"/></a>
+        <a href="images/image-6.jpg"><img src="images/image-6.jpg"/></a>
+        <a href="images/image-7.jpg"><img src="images/image-7.jpg"/></a>
+        <a href="images/image-8.jpg"><img src="images/image-8.jpg"/></a>
+        <a href="images/image-9.jpg"><img src="images/image-9.jpg"/></a>
+        <a href="images/image-10.jpg"><img src="images/image-10.jpg"/></a>
+        <a href="images/image-11.jpg"><img src="images/image-11.jpg"/></a>
+        <a href="images/image-12.jpg"><img src="images/image-12.jpg"/></a>
+        <a href="images/image-13.jpg"><img src="images/image-13.jpg"/></a>
+        <a href="images/image-14.jpg"><img src="images/image-14.jpg"/></a>
+        <a href="images/image-15.JPG"><img src="images/image-15.JPG"/></a>
+        <a href="images/image-16.jpg"><img src="images/image-16.jpg"/></a>
+      </div>
+    </div> 
+  </body>
```

Comment:
Can you add a closing html tag here? As of the HTML5 specification, you can also omit it. I'll leave it up to you.

ref: https://google.github.io/styleguide/htmlcssguide.html , https://html.spec.whatwg.org/multipage/syntax.html#syntax-tag-omission

Reply:
Done!